libs/dh/imbalance-prices/shell/src/lib/table-day-view/dh-price-color.directive.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Directive, ElementRef, inject, Input, Renderer2 } from '@angular/core';
+
+import { WattColorHelperService, WattColor } from '@energinet-datahub/watt/color';
+
+@Directive({
+  selector: '[dhPriceColor]',
+  standalone: true,
+})
+export class DhPriceColorDirective {
+  private readonly elementRef = inject(ElementRef);
+  private readonly renderer = inject(Renderer2);
+  private readonly wattColorService = inject(WattColorHelperService);
+
+  @Input() set dhPriceColor(value: number | undefined) {
```

Comment:
Suggestion: Would it not be better to use input signal or a transform function?

Reply:
Of cause you do mutate something, so maybe not

Reply:
I was thinking of using a Signal input but the value comes from an API, and it's not wrapped in a Signal. So to wrap it in a Signal just to send it in ... maybe ok, but probably unnecessary.

After opening the PR, I thought of another solution - toggling a class in the `dh-table-day-view` component instead of creating a directive. This seems simpler, so I pushed a new commit.